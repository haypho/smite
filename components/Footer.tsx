import { Box, Container, Typography } from "@mui/material";

export const Footer = () => (
  <footer>
    <Container>
      <Box
        display="flex"
        alignItems="center"
        justifyContent="center"
        paddingY={4}
      >
        <Typography>
          Data provided by Hi-Rez Studios. Â© Hi-Rez Studios, Inc. All rights
          reserved.
        </Typography>
      </Box>
    </Container>
  </footer>
);
