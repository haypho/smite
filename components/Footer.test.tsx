import { Footer } from "./Footer";
import { render } from "@testing-library/react";

describe("<Footer />", () => {
  it("should match the snapshot", () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
