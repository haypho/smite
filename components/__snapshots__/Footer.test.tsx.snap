// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match the snapshot 1`] = `
<div>
  <footer>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-1vom740"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Data provided by Hi-Rez Studios. Â© Hi-Rez Studios, Inc. All rights reserved.
        </p>
      </div>
    </div>
  </footer>
</div>
`;
